client/src/templates/Challenges/classic/show.tsx

```diff
@@ -217,6 +220,7 @@ function ShowClassic({
 }: ShowClassicProps) {
   const { t } = useTranslation();
   const [resizing, setResizing] = useState(false);
+  const [prefetched, setPrefetched] = useState(false);
```

Comment:
```suggestion
```
We don't need this if the effect only triggers when `nextChallengePath` changes.